### PR TITLE
feat: add Hubble brand

### PR DESCRIPTION
**Pull request by americano98**
This pull request brings branding for Hubble.

Steps to check:

- open [preview](https://deploy-preview-380--cilium.netlify.app/brand/)
- make sure that everything looks and feels as expected

### CODE REVIEW

**Review by prosazhin — APPROVED** [review]

File: src/components/pages/brand/logos/logos-row/logos-row.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <ul className="gap-grid mt-4 grid grid-cols-12 gap-y-5">
+      {logos.map(({ src, width, height }, index) => (
+        <li
+          className={classNames(
+            'col-span-full flex h-[192px] items-center justify-center rounded-xl px-11 sm:col-span-6 lg:col-span-3',
+            isDark ? 'bg-gray-7' : 'bg-gray-4'
+          )}
+          key={index}
+        >
+          <img className="max-w-none shrink-0" src={src} width={width} height={height} alt="" />
```
Comment (prosazhin) [inline src/components/pages/brand/logos/logos-row/logos-row.jsx:17]:
```suggestion
          <img className="max-w-none shrink-0" src={src} width={width} height={height} alt="" loading="lazy" />
```

**Review by prosazhin — APPROVED** [review]

**Comment by prosazhin** [comment]
@xmulligan Hello. Please approve and merge this pull request.

**Comment by americano98** [comment]
Hi @xmulligan, we don't have sufficient rights to merge in this repository. Could you please grant us access?

**Comment by xmulligan** [comment]
Thanks, I like to review everything before it is merged

**Review by xmulligan — APPROVED** [review]

**Pull request merged**